webpack/assets/javascripts/react_app/components/notifications/NotificationsDrawerHelpers.js

```diff
@@ -0,0 +1,10 @@
+export const formatDateTime = datetime =>
+  new Date(datetime).toLocaleDateString('en-US', {
```

Comment:
should use `const locale = documentLocale().replace(/-/g, '_');` instead of `en-US`